Xamarin.Forms.Core/TouchGestureRecognizer.cs

```diff
@@ -0,0 +1,223 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+
+namespace Xamarin.Forms
+{
+	public class TouchGestureRecognizer : GestureRecognizer
```

Comment:
is this meant to be abstract, or how do you plan to use it ? (I see no commands, for example)

Reply:
I am thinking of using this class as a base class for custom gesture recognizers.
Because Since this class can work with approximately 8 states for each touch, I thought that doing 8 commands does not make sense.
And commands can be implemented in successor classes.
You can see it in other gesture recognizers.

Reply:
done